lib/api/web-element/commands/sendKeys.js

```diff
@@ -22,10 +22,10 @@
  * @instance
  * @syntax browser.element(selector).sendKeys(...keys)
  * @param {...string} keys
- * @see https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebElement.html#sendKeys
+ * @link /#element-send-keys
  * @returns {ScopedWebElement}
  */
-module.exports.command = function(...args) {
+module.exports.command = function (...args) {
```

Comment:
I suggest, remove this unnecessary space between as it shows no significant change.